{"uuid": "d824dd7e-03bd-4fd4-a52e-cc3474981e70", "children": ["8ec175af-7076-44f3-91d7-2f5c23e7c075", "832c4924-fc0f-4584-8889-964951e228b1"], "befores": [{"name": "test_setup", "status": "broken", "statusDetails": {"message": "ModuleNotFoundError: No module named 'webdriver_manager'\n", "trace": "  File \"/Users/bhargavsrinivasan/Library/Python/3.8/lib/python/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/bhargavsrinivasan/Library/Python/3.8/lib/python/site-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/bhargavsrinivasan/Library/Python/3.8/lib/python/site-packages/_pytest/fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/bhargavsrinivasan/python-pytest-allure/conftest.py\", line 16, in test_setup\n    from webdriver_manager.chrome import ChromeDriverManager\n"}, "start": 1650036378035, "stop": 1650036378036}], "start": 1650036378035, "stop": 1650036378112}